### ISSUE TITLE
Error on encoding x-IBM1098 and x-IBM1097

### ISSUE THREAD

**Issue by sgsm74** [pr0gramista/charset_converter#21]
When I want to try encode Persian character it show same digit for all characters
**Uint8List encoded = await CharsetConverter.encode("x-IBM1098", "سلام");**
It shows: **127, 127, 127, 127, 127**
It's so weird a digit for different character and when I try to decode this numbers It shows some square icon.

**Comment by pr0gramista** [comment]
Hmm, I could not find x-IBM1098. What platform do you use?

I check if ISO-8859-6 would work and it seems fine.

**Comment by sgsm74** [comment]
I saw it on availableCharacterSets and It's the only characterset that works fine with Persian language and It's for Iran System Encoding.
Altough the decode function work fine with that.
I used it on Android.

**Comment by pr0gramista** [comment]
Alright, I'll look into this, but I need to get some Androids first.

If you have any clues why that might happen I'd be happy to learn that. Thanks!

**Comment by pr0gramista** [comment]
I am not sure if this text can be expressed in this encoding and it seems like this encoding returns 127 when it meets a character it can not understand.
I was successful with [IBM-1256/CP-1256/Windows-1256](https://en.wikipedia.org/wiki/Windows-1256) for your example text I got bytes 211, 225, 199, 227 where ISO-8859-6 returned 211, 228, 199, 229.

@sgsm74 have you perhaps already solved this issue?

**Comment by sgsm74** [comment]
Yep, by creating my own class for converting characters. I used it for an inbuilt-printer pos device (citaq) and It was okay
